presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -138,7 +138,6 @@
 
 import javax.annotation.Nullable;
 import javax.inject.Inject;
-
```

Comment:
This blank line between `java` and `javax` is intentional. You can find out code style with settings for IDEA here: https://github.com/airlift/codestyle